src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -41,6 +41,8 @@ class Definition
     private $synchronized = false;
     private $lazy = false;
     private $decoratedService;
+    private $autowired = false;
+    private $autowiringTypes = array();
```

Comment:
`$definition->isAutowire()` looks weirder than `$definition->isAutowired()`. It's why I've kept `autowired` here but not in the loaders.